<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiLevelBox</class>
 <widget class="QDialog" name="MultiLevelBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(2, 2, 2);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 132, 10);
background-color: rgb(255, 255, 255);</string>
     </property>
     <column>
      <property name="text">
       <string>Level</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="DialogueAttribute">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0.7);</string>
        </property>
        <item>
         <property name="text">
          <string>--Select Attribute--</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Title</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Certificate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Director</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rating</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Duration</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Votes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Genre</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>179</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="AddBtn">
        <property name="font">
         <font>
          <family>Kristen ITC</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton:hover{
background-color:rgb(255, 132, 10);
color: rgb(0,0,0);
}

QPushButton{
background-color:rgba(255,255,255,0.2);
color:rgb(255, 132, 10);
}</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteBtn">
        <property name="font">
         <font>
          <family>Kristen ITC</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton:hover{
background-color:rgb(255, 132, 10);
color: rgb(0,0,0);
}

QPushButton{
background-color:rgba(255,255,255,0.2);
color:rgb(255, 132, 10);
}</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="doneBtn">
        <property name="font">
         <font>
          <family>Kristen ITC</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton:hover{
background-color:rgb(255, 132, 10);
color: rgb(0,0,0);
}

QPushButton{
background-color:rgba(255,255,255,0.2);
color:rgb(255, 132, 10);
}</string>
        </property>
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>doneBtn</sender>
   <signal>clicked(bool)</signal>
   <receiver>MultiLevelBox</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
